### PR TITLE
Cast spaces

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | /
| Type?         | improvement
| Category?     | CO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | /
| How to test?  | Nothing to test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11967)

### CODE REVIEW

File: admin-dev/functions.php [hunk 1]
```diff
@@ -463,7 +463,7 @@ function runAdminTab($tab, $ajax_mode = false)
             if (!$ajax_mode && Shop::isFeatureActive() && Shop::getContext() != Shop::CONTEXT_ALL && Context::getContext()->controller->multishop_context != Shop::CONTEXT_ALL) {
                 echo '<div class="multishop_info">';
                 if (Shop::getContext() == Shop::CONTEXT_GROUP) {
-                    $shop_group = new ShopGroup((int)Shop::getContextShopGroupID());
+                    $shop_group = new ShopGroup((int) Shop::getContextShopGroupID());
                     printf(Translate::getAdminTranslation('You are configuring your store for group shop %s'), '<b>'.$shop_group->name.'</b>');
```
Comment (LouiseBonnard) [inline admin-dev/functions.php:467]: Shouldn't it be localized in the 1.7 catalogs?

File: admin-dev/functions.php [hunk 2]
```diff
@@ -463,7 +463,7 @@ function runAdminTab($tab, $ajax_mode = false)
...
                 } elseif (Shop::getContext() == Shop::CONTEXT_SHOP) {
                     printf(Translate::getAdminTranslation('You are configuring your store for shop %s'), '<b>'.Context::getContext()->shop->name.'</b>');
```
Comment (LouiseBonnard) [inline admin-dev/functions.php:469]: Same question here!

**Comment by matks** [comment]
Dear @MathiasReker, as discussed together and following our [Contributing rules](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requests-requirements) I must unfortunately close this PR as it is too large to be thoroughly reviewed. Thank you for the contribution.

**Comment by MathiasReker** [comment]
@matks No problem :-)

**Pull request closed without merging**